lib/rubocop/cop/lint/ambiguous_block_association.rb

```diff
@@ -26,7 +28,14 @@ module Lint
       #   # good
       #   # Lambda arguments require no disambiguation
       #   foo = ->(bar) { bar.baz }
+      #
+      # @example IgnoredMethods: [change]
+      #
+      #   # good
+      #   expect { do_something }.to change { object.attribute }
       class AmbiguousBlockAssociation < Base
+        include IgnoredMethods
```

Comment:
Ah, I had forgotten the mixing was named like this. You can escape from legacy. :D